Containerfile.aws-load-balancer-operator

```
@@ -0,0 +1,29 @@
+# Detect the drift from the upstream Dockerfile
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS drift
+WORKDIR /app
+COPY drift-cache/Dockerfile Dockerfile.cached
+COPY Dockerfile .
+# If the command below fails it means that the upstream Dockerfile changed.
+# You have to update the Konflux Containerfile accordingly.
+# drift-cache/Dockerfile can be updated with the upstream contents once the Konflux version is aligned.
+RUN [ "$(sha1sum Dockerfile.cached | cut -d' ' -f1)" = "$(sha1sum Dockerfile | cut -d' ' -f1)" ]
+
+
+FROM registry.access.redhat.com/ubi8/go-toolset:1.22 as builder
+# dummy copy to trigger the drift detection
+COPY --from=drift /app/Dockerfile.cached .
+COPY . /workspace
+WORKDIR /workspace
+RUN git config --global --add safe.directory /workspace
+# Build
+RUN GOOS=linux GOARCH=amd64 go build -tags strictfipsruntime -a -o /usr/bin/manager main.go
+
+FROM registry.redhat.io/rhel8-6-els/rhel:8.6-1737
+LABEL maintainer="Red Hat, Inc."
+LABEL com.redhat.component="aws-load-balancer-operator-container"
+LABEL name="aws-load-balancer-operator"
+LABEL version="${CI_VERSION}"
```

`${CI_VERSION}` placeholder won't be filled, we can remove set it manually to the latest version released: `1.2.0`.
```suggestion
LABEL version="1.2.0"
```